src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -613,12 +614,10 @@ public override Assembly GetSatelliteAssembly(CultureInfo culture, Version versi
             return InternalGetSatelliteAssembly(culture, version, true);
         }
 
-        [System.Security.DynamicSecurityMethod] // Methods containing StackCrawlMark local var has to be marked DynamicSecurityMethod
```

Comment:
This cannot be removed. It is enforced on checked builds for every method that has `StackCrawlMark` inside.